### ISSUE TITLE
Show changes in the code corresponding to a change in the test result

### ISSUE THREAD

**Issue by plexoos** [BNLNPPS/BirdView#1]
It would be really cool to have a feature implemented when one can click on a data point and see the commits/changes in the code submitted on that date.

![UU193_embed](https://user-images.githubusercontent.com/5005079/80753086-aac95600-8afa-11ea-8b2b-82605c29b025.png)

**Comment by amolhj** [comment]
Hello, Thank you for your suggestion. However, if you see the commits from CVS (https://www.star.bnl.gov/webdata/cvs/commitHistory.html). Someday, there are many commits. Therefore, it will be inconvenient to show all on a single point.

**Comment by plexoos** [comment]
Yes, the patch can be big or it can be small but in either case to see the reason for a jump in a variable is better then nothing. How do you suggest to identify the cause for a change in the plot?

**Comment by amolhj** [comment]
BirdView does not have capability to identify the cause of change. It is monitoring system which shows the changes effectively. For example, if the changes are shown by BirdView, then we can see on CVS commit history on that day what has changed? Then, based on developer's knowledge, they can identify what is the cause of this change. CVS commit history page will give details of all commits as well as what has changed too.

**Comment by plexoos** [comment]
So, basically you are suggesting to do the same thing as I initially proposed, i.e. one needs to see the change in the code to establish the reason for a jump in the plot. But you suggest to do it manually by noting the date of the change then logging into a RACF machine, issue CVS commands, and see the patch. I thought this tool could help automate all of this.

By the way, this web page (https://www.star.bnl.gov/webdata/cvs/commitHistory.html) does not show the differences in the code either when you click on the listed files.

**Comment by amolhj** [comment]
This (https://www.star.bnl.gov/webdata/cvs/commitHistory.html) is webpage. Why do you have to login on RACF machine and issue CVS command?
Try to open commit history webpage by using protected credentials. Then, you can see the list of files updated on a specific date. Open any one of them and then click on diff to ....., which will give you difference between older version and newer version.
